src/tasks/includes/utilities.js

```diff
@@ -13,14 +15,35 @@ const Promise = require('bluebird');
 const utilities = {
 
   /**
-   * Generic error handler for streams called in `watch` tasks (used by gulp-plumber)
+   * Handles the output for any errors that might have been captured
+   * during the build and zip Gulp tasks.
+   *
+   * @memberof slate-cli.utilities
+   */
+  outputErrors: () => {
+    if (!errors.length) {
```

Comment:
I generally try to do `errors.length =< 0`  on `.length` instead of assuming type conversion to boolean. That's a personal preference thing but I thought I'd comment anyways.